test/Sleep-test.js

```diff
@@ -103,25 +148,41 @@ describe("Sleep", () => {
   });
 
   it("should return users average number of hours slept per day", () => {
-    sleep.userSleepData(sampleSleepData, 1)
-    expect(sleep.userSleep).to.be.deep.equal(filterUser1);
+    expect(sleep.userSleepData(1)).to.be.deep.equal(filterUser1);
   });
 
-  it("should return users average number of hours slept per day", () => {
-    sleep.userSleepData(sampleSleepData, 2)
-    expect(sleep.userSleep).to.be.deep.equal(filterUser2);
+  it("should return another users average number of hours slept per day", () => {
+    expect(sleep.userSleepData(2)).to.be.deep.equal(filterUser2);
   });
 
-  it("should return average all-time daily sleep per a user", () =>{
-    expect(sleep.averageAllTimeSleep()).to.be.equal(2.6)
+  it("should return average all-time daily sleep for a user", () =>{
+    expect(sleep.averageAllTimeSleep(1)).to.be.equal(2.6)
   });
 
   it("should should return sleep for a specified day", () =>{
-    expect(sleep.daySleep("2019/06/16")).to.equal(4.1)
+    expect(sleep.daySleep("2019/06/16", 1)).to.equal(4.1)
   });
 
   it("should return daily sleep over 7 day period", () => {
-    expect(sleep.dailySleepPerGivenWeek("2019/06/15")).to.deep.equal(weeklyData)
+    expect(sleep.weeklySleepProperties("2019/06/15", 1, 'hoursSlept')).to.deep.equal(hoursSlept)
   });
 
-});
\ No newline at end of file
+  it("should return daily sleep quality over 7 day period", () => {
+    sleep.userSleep = sampleSleepData;
+    expect(sleep.weeklySleepProperties("2019/06/15",  2, 'sleepQuality')).to.deep.equal(sleepQuality)
+  })
+
+  it("should return average sleep quality for all users", () => {
+    expect(sleep.averageSleepQuality()).to.be.equal(3.1)
+  })
+
+  it("should return users that average sleep quality above 3", () => {
+    sleep.userSleep = sampleSleepData;
+    expect(sleep.sleepQualityAboveThree("2019/06/15")).to.deep.equal([2])
+  })
+
+  // it("should find the user who slept the most for a specified day", () =>{
+  //   expect(sleep.userWhoSleptTheMost("2019/06/15", sampleSleepData)).to.deep.equal(7)
+  // })
+
```

Comment:
Adjusted these a bit to use properties and IDs, you can see most tests reference ID now.

Reply:
looks good